roles/scripts/files/wp-list-env

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+##
+# Gives user easy way to environment variables which are important for him
+##
+
+# Source environment if doesn't exist yet
+if [ -z $PS1 ]; then
+  source /etc/container_environment.sh
+fi
+
+IFS='%' read -a envvars <<< "$LIST_CONTAINER_ENVS"
```

Comment:
is this tested? i don't think the vagrant box has /etc/container_environment since it's not a container